pkg/asset/installconfig/ovirt/validation.go

```diff
@@ -125,3 +126,22 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+// ValidateProvisioning - Perform Provisioning Validations for oVirt
+func ValidateProvisioning(ic *types.InstallConfig) error {
+	clusterName := ic.ObjectMeta.Name
+	zoneName := ic.BaseDomain
+	validFqdns := []string{
+		fmt.Sprintf("api.%s.%s", clusterName, zoneName),
+		fmt.Sprintf("test.apps.%s.%s", clusterName, zoneName),
+	}
+
+	for _, addr := range validFqdns {
+		_, err := net.LookupIP(addr)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I think we should be returning better error messages.

Reply:
done